libminifi/include/io/CRCStream.h

```diff
@@ -27,6 +27,7 @@
 #include <cassert>
 
 #ifdef WIN32
+#pragma warning(disable : 4250)
```

Comment:
Could you please add a comment about what this does or prevents?

Reply:
Updated in be9cace2da1aa161129ac9c22986033dafc199db